code/modules/research/nanites/nanite_programs/sensor.dm

```diff
@@ -210,11 +210,12 @@
 		if(CLONE)
 			damage_amt = host_mob.getCloneLoss()
 
-	if(damage_amt >= damage.get_value())
-		if(check_above)
-			reached_threshold = TRUE
-	else if(!check_above)
-		reached_threshold = TRUE
+	if(check_above)
+		if(damage_amt >= damage.get_value())
+				reached_threshold = TRUE
+	else
+		if(damage_amt < damage.get_value())
+				reached_threshold = TRUE
```

Comment:
Same as in line 215, remove one tab.

Reply:
ugh sorry weird about the indentation.  Fixing now